Gemfile

```diff
@@ -99,6 +101,7 @@ group :development, :test do
   gem 'guard-rspec', '~> 4.5.0', require: false
   gem 'coveralls', '~> 0.7.3', require: false
   gem 'timecop'
+  gem 'byebug'
```

Comment:
We already have `pry` -- would rather not add another. But if it is really needed, put it in the `development` section.